README.md

```diff
@@ -28,22 +28,22 @@ endpoint. The first is through an environment variable:
 The second is directly on the command line:
 
 ```
-    $ python openshift-usage-scripts/openshift_prometheus_metrics.py --openshift-url <prometheus url>
+    $ python -m openshift-usage-scripts.openshift_prometheus_metrics --openshift-url <prometheus url>
```

Comment:
This should be `openshift_metrics.openshift_prometheus_metrics` instead of `openshift-usage-scripts.openshift_prometheus_metrics`

A missed typo from long ago?

Reply:
correct, I have updated it every where else. thanks!